README.md

```diff
@@ -705,3 +706,17 @@ npm run docs
 **Contributing**
 
 Contributions welcome! Please use GitHub issues for suggestions/concerns - if you prefer to express your intentions in code, feel free to submit a pull request.
+
+## Diagnosing Common Issues
+
+* Is the `expirationTime` in future?  If no, change it to a time in the future.
+
+* Is the `expirationTime` a fractional second?  If yes, round the listing time to the nearest second.
+
+* Are the input addresses all strings? If no, convert them to strings.
+
+* Are the input addresses checksummed or lowercased?  You might need to use the checksummed version of the address.
```

Comment:
I thought it worked best if we lowercased? we should fix any areas that don't convert to lowercase